open-hackathon-client/src/client/static/js/views/oh.manage.hostserver.js

```diff
@@ -0,0 +1,252 @@
+        $('#private_ip').val("");
+        $('#private_docker_api_port').val("4243");
+        $('#container_max_count').val("1");
+    }
+
+    function getFormData() {
+        data = {};
+        data.vm_name = $('#vm_name').val();
+        data.public_dns = $('#public_dns').val();
+        data.public_ip = $('#public_ip').val();
+        data.public_docker_api_port = $('#public_docker_api_port').val();
+        data.private_ip = $('#private_ip').val();
+        data.private_docker_api_port = $('#private_docker_api_port').val();
+        data.container_max_count = parseInt($('#container_max_count').val(), 10);
+        data.disabled = $('#disabled').val() == '启用'? 0:1; //0:abled 1:disabled
+        data.is_auto = 2 //auto=1, manual=2;
+        return data;
+    }
+
+    function createHostServer(data) {
+        alert("该请求需要测试Host Server的Docker服务是否可用,点击确认/ok开始，请等待最多5秒");
+        return oh.api.admin.hostserver.post({
+            header: {
+                hackathon_name: currentHackathon
+            },
+            body: data
+        }, function (response) {
+            if (response.error) {
+                oh.comm.alert('错误', data.error.friendly_message);
+            } else {
+                clearFormData();
+                toggleTable();
+                pageLoad();
+                $("#submit_button").removeAttr("disabled")
+            }
+        });
+    }
+
+    function updateHostServer(data) {
+        alert("该请求需要测试Host Server的Docker服务是否可用,点击确认/ok开始，请等待最多5秒");
+        return oh.api.admin.hostserver.put({
+            header: {
+                hackathon_name: currentHackathon
+            },
+            body: data
+        }, function(response){
+            if(response.error) {
+                oh.comm.alert('错误', response.error.friendly_message);
+            } else{
+                clearFormData();
+                toggleTable();
+                pageLoad();
+                $("#submit_button").removeAttr("disabled")
+            }
+        });
+    }
+
+    function deleteHostServer(host_server_id) {
+        return oh.api.admin.hostserver.delete({
+            header: {hackathon_name: currentHackathon},
+            query: {id: host_server_id}
+        }, function(response) {
+            if(response.error){
+                oh.comm.alert('错误', response.error.friendly_message);
+            }else{
+                pageLoad();
+            }
+        });
+    }
+
+    function refreshHostServerContainerCount(host_server_id) {
+        var hostServerInfo = oh.api.admin.hostserver.get({
+            header: {hackathon_name: currentHackathon},
+            query: {id: host_server_id}
+        },function(response){
+            if(response.error){
+                oh.comm.alert('错误', response.error.friendly_message);
+            }else{
+                pageLoad();
+            }
+        });
+        return hostServerInfo.container_count;
+    }
+
+    // initial table to show admin list
+    function pageLoad() {
```

Comment:
please rename this method to `reload_host_servers` or similar. `pageLoad()` usually means the first time the whole page is loaded including all initial scripts. In this page,`init()` method is in fact `pageload()` while `pageload` which are called multiple times contains actually part of initialization.